### ISSUE TITLE
Share recipes between boxes

### ISSUE THREAD

**Issue by JbIPS** [tariqbenezza/myRecipeBox#1]
# Enhancement proposal

When visiting a box
As a connected user not owning the box
I should be able to add a recipe to my box

**Comment by tariqbenezza** [comment]
What do you suggest @JbIPS ?

- I'm logging in to my box
- I save it like a kind of session 
- I'm going to another box
- On each recipe I propose to add it to my own box

**Comment by JbIPS** [comment]
Exactly! And yes you should store the session in a cookie so you can detect your in another box, thus showing the add button
